internal/commands/create.go

```diff
@@ -252,6 +243,15 @@ func setKindMatcher(constraint *unstructured.Unstructured, kindMatchers []rego.K
 	return nil
 }
 
+func appendIfNotExists(currentItems []interface{}, newItem string) []interface{} {
```

Comment:
I think this sort of helper method makes sense. I would change it to just taking a `[]string` rather than a `[]interface` as I only see this ever needing to work with strings. If we _do_ end up needing more, we can cross that road when we get there.

It also may be a good idea to compare with `EqualFold` in the event that someone types `ingress` or `Ingress`. Looks like that is a bug that could exist today.

Reply:
I think it's good.
However, `SetNestedSlice` get panic when be passed `map[string][]string` as value,
so I'll modify `setKindMatcher` to create `[]string` slice and then convert it to `[]interface{}`.